environment.yml

```diff
@@ -43,3 +43,4 @@ dependencies:
   - pip:
     - sphinx-toolbox
     - git+https://github.com/OpenFreeEnergy/gufe@main
+    - git+https://github.com/openforcefield/openff-models.git@fix-openmm
```

Comment:
I can merge without this 😅

Reply:
I committed a feature to my fork's `main` prior to making this branch ... tsk tsk